charts/pgadmin4/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: pgAdmin4 is a web based administration tool for PostgreSQL database
 name: pgadmin4
-version: 1.3.8
+version: 1.3.9
```

Comment:
Since this PR is about an addition of new parameter, semantically it should be next minor release `1.4.0`.

Reply:
updated.